configs/small.yaml

```diff
@@ -98,6 +99,7 @@ model:                              # specify your model architecture here
             embedding_dim: 16
             scale: False
             freeze: False           # if True, embeddings are not updated during training
```

Comment:
Have you tried loading the fasttext embeddings and then freezing them for IWSLT?

Reply:
No, I haven't tried freezing. I've specified neither "scale" nor "freeze" option in config, that is, default values (scale: False, freeze: False) are used, so I just initialized the WE by the pre-trained weights, nothing more. Do you simply want to see the results with freezing? Or have I made any mistake, i.e. I should always freeze WE when I initialize it by pretrained weights??

Reply:
I was just curious about results :) no mistakes.

Reply:
I was also thinking that renormalization with the randomly initialised OOV words would be good so that all embeddings are on the same scale, but we can keep for later. It might be worth making it a default if it turns out to be bénéficient.

Reply:
oh I see. Maybe we can at least raise attention in doc: loading pretrained weights may cause scaling issue, users should be careful on choosing WE initialization.

I'll run extra experiments with freezing and report results here, when we have free space in our GPUs. (Recently, our gpus are occupied...)

Reply:
yes that sounds like a very good solution. Don't worry about the experiments. What do you think about making a small tutorial of it? With exactly the experiments you ran so far -- I think it's worth having an extra page in the documentation for it (like the toy task), since it's a common use case.